js/src/ui/adminBar.js

```diff
@@ -1,14 +1,29 @@
 /**
- * Updates the traffic light present on the page
+ * Updates the admin bar for the page.
  *
  * @param {Object} indicator The indicator for the keyword score.
+ * @param {string} keyword The focus keyphrase for the post.
  *
  * @returns {void}
  */
-function updateAdminBar( indicator ) {
+function updateAdminBar( indicator, keyword ) {
+	// Updates the traffic light
 	jQuery( ".adminbar-seo-score" )
 		.attr( "class", "wpseo-score-icon adminbar-seo-score " + indicator.className )
 		.find( ".adminbar-seo-score-text" ).text( indicator.screenReaderText );
+
+	// Updates the focus keyhprase in the menu
+	const keywordMenuElement = jQuery( "#wp-admin-bar-wpseo-focus-keyphrase" );
+
+	if ( keyword === "" ) {
+		keywordMenuElement.hide();
+	}
+
+	if ( keyword !== "" && typeof keyword !== "undefined" ) {
+		keywordMenuElement
+			.show()
+			.find( ".wpseo-focus-keyphrase-ab-item" ).text( keyword );
+	}
```

Comment:
I'd probably consider to refactor a bit the logic of the conditionals.

1
not sure if there are cases where keyword can be actually `undefined` but if there are it should probably be checked early even before the first `if` at line 18 and return.

2
when emptying the keyphrase field, the menu item element isn't fully emptied: the first letter stays there: not a big issue because the element gets hidden but I'd probably update the element text in any case, whether keyword is empty or not. E.g. by moving `keywordMenuElement.find( ".wpseo-focus-keyphrase-ab-item" ).text( keyword )` before the first `if`.

Screenshot: emptied field but the menu item still has the first keyphrase letter:

<img width="737" alt="Screenshot 2019-10-09 at 11 05 44" src="https://user-images.githubusercontent.com/1682452/66469309-e5e53880-ea87-11e9-88a7-96e76f33f2ba.png">


3
With the above implemented, the first `if` could just hide the element and return. The second `if` could be removed and the code could just show the element.